mala/descriptors/bispectrum.py

```diff
@@ -196,6 +203,6 @@ def _calculate(self, atoms, outdir, grid_dimensions, **kwargs):
             # it does not necessarily do that - so we have do go down
             # that route.
             if self.parameters.descriptors_contain_xyz:
-                return snap_descriptors_np.copy(), nx*ny*nz
+                return snap_descriptors_np, nx*ny*nz
             else:
-                return snap_descriptors_np[:, :, :, 3:].copy(), nx*ny*nz
```

Comment:
The comment above says that the copy() is needed, but this PR removes it.

Reply:
The copy has been moved in the lammps_util.py file, but you are right, the comment should be adapted. Will do that.

Reply:
Fixed by 2c7cf122acf4eca51f632def689b93414de177a5